context.md

```diff
@@ -272,23 +282,94 @@ var instructor = {
 instructor.displayFoods();
 ```
 
-**Mini-exercise**: Use what we know about scope to explain why this works.
-
 ## Exercise: [Cat Surprise](https://github.com/ga-wdi-exercises/cat-surprise)
```

Comment:
yeah honestly not sure... maybe even a little exercise like what do you expect the value of this to be... in multiple contexts